latest/index.bs

```diff
@@ -177,308 +177,99 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
-            |       |             # The rows in each obsm array SHOULD be chunked the same as the columns in X.
-            |       ├── .zarray
-            |       ├── 0.0
-            │       │   ...
-            │       └── n.m
-            |
-            ├── obsp              # You MAY add a obsp group comtainer. The obsp group contains sparse arrays that annotate the rows in X.
-            |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
-            |   |
-            │   ├── .zgroup
-            |   |
-            |   ├── .zattrs       # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
-            |   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
-            |   |                 # `.zattrs` MUST contain `"keys"`, which is an array of the names of the subgroups containing `obsp` arrays.
-            |   │
-            │   └── obsp_0        # You MAY add a zarr group for each obsp array.
-            |       |             # Each obsp array MUST have the same number of rows as rows in X.
-            |       |
-            │       ├── .zgroup
-            |       |
-            |       ├── .zattrs   # `.zattrs` MUST contain `"encoding-type"`, which is set to `"csr_matrix"` or `"csc_matrix"` for compressed sparse row and compressed sparse column, respectively.          
-            |       │             # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
-            |       |             # `.zattrs` MUST contain `"shape"` which is an array giving the shape of the densified array.
-            |       |
-            |       ├── data      # You MUST add a one-dimensional zarr array named "data". 
-            |       |   |         # `data` MAY be chunked as the user desires.
-            |       |   ├── .zarray
-            |       |   |
-            |       |   ├── 0
-            │       │   |   ...
-            │       |   └── n
-            |       |
-            |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
-            |       |   |         # `indices` MAY be chunked as the user desires.
-            |       |   ├── .zarray  # `indices` MUST be an `int` dtype.
-            |       |   |
-            |       |   ├── 0
-            │       │   |   ...
-            │       |   └── n
-            |       |
-            |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
-            |           |         # `indptr` MAY be chunked as the user desires.
-            |           ├── .zarray  # `indptr` MUST be an `int` dtype.
-            |           |
-            |           ├── 0
-            │           |   ...
-            │           └── n
+            ├── .zgroup
+            ├── .zattrs
```

Comment:
What goes in this `tables/.zattrs` file?
Does this list the child tables in any way? E.g. `{"tables": ["table1", "anotherTable"]}` in the same way that `labels/.zattrs` does? https://ngff.openmicroscopy.org/0.4/index.html#labels-md
This is really essential if you can't browse the subgroups.

Reply:
I included this suggestion in a [first round of including feedback](https://github.com/minnerbe/ngff/commit/15c9a2001b48db7573cb7d1f58213d89d5348927). How does it look to you?

Reply:
Thanks, that looks good from a practical point of view (allows me to know what tables exist etc).
However, it's a bit different from what I was expecting, which was just to list the single table `tables.attrs["tables"] = ["anndata",]` and for the tables spec to define what the sub-tables are called. So I guess that highlights what you're proposing in that the spec doesn't define what any of the sub-tables are called (in the way that AnnData does) so you're free to name them anything? E.g. `var` or `obs` could be `col_info` or `extra` etc?

Reply:
Exactly, I want to keep it as generic as possible until there is a very compelling argument to do otherwise.